MAUI/DataForm/layout.md

```diff
@@ -0,0 +1,750 @@
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Layout in Xamarin DataForm (SfDataForm)
+
+## Overview
+
+The data form supports linear and grid layouts and manages layout of label, editor, and validation label.
+
+## Linear layout support
+
+By default, the data form arranges the fields one-by-one. It is applicable for both label positions: left and top.
+
+When the label position is Left, the linear layout is shown as follows:
+
+When the label position is Top, the linear layout is shown as follows:
+
+## Grid layout support
+
+By default, the data form arranges one data field per row. It is possible to have more than one data fields per row by setting the [ColumnCount] property which provides grid like layout for the data form. The order of data form field can also be changed in a row using [ItemsOrderInRow]
+
+{% tabs %}
+{% highlight xaml %}
+
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:DataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+             x:Class="CommitMode.MainPage">
+
+    <DataForm:SfDataForm
+        x:Name="DataForm"
+        ColumnCount="2"
+        >
+    </DataForm:SfDataForm>
+
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# %}
+
+this.DataForm.ColumnCount = 2;
+
+{% endhighlight %}
+{% endtabs %}
+
+N> Setting the `ColumnCount` property to SfDataForm does not arrange the data field in a group according to the column count. To set the column count for data fields in the data form group, refer to [loading different layout for data form group]
+
+When the label position is Left, the grid layout is shown as follows:
+
+When the label position is Top, the grid layout is shown as follows:
+
+## Label visibility
+
+The label can be hided by defining the [DataFormDisplayOptions] attribute or by handling `GenerateDataFormItem` event. In this case, only the editor will be loaded.
+
+### Using attributes
+
+{% tabs %}
+{% highlight c# %}
+private double? percentage;
+
+    [DataFormDisplayOptions(ShowLabel = false)]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+### Using event
+
+{% tabs %}
+{% highlight c# %}
+
+ this.DataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name")
+        {
+            e.DataFormItem.PlaceholderText = "Enter your name";
+            e.DataFormItem.ShowLabel = false;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Label position
+
+Label can be positioned either top or left side of the editor. By using the [DataFormDefaultLayoutSettings.LabelPosition] property, you can layout the label associated with editor.
```

Comment:
you can layout the label associated with editor for data form

Reply:
changed content